columnphysics/icepack_therm_vertical.F90

```diff
@@ -378,9 +378,9 @@ subroutine thermo_vertical (dt,          aicen,     &
       Tsnice = c0
       if ((hslyr+hilyr) > puny) then
          if (hslyr > puny) then
```

Comment:
If `hslyr > puny`, then `hslyr+hilyr > puny` since `hilyr >= 0`, so then is the outer conditional necessary?  It seems redundant, logically.

Reply:
Good catch.